src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx

```diff
@@ -3,19 +3,69 @@ import ManageReleaseContext, {
 } from '@admin/pages/release/ManageReleaseContext';
 import Tabs from '@common/components/Tabs';
 import TabsSection from '@common/components/TabsSection';
-import React, { useContext } from 'react';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import footnotesService from '@admin/services/release/edit-release/footnotes/service';
+import permissionService from '@admin/services/permissions/service';
+import { generateFootnoteMetaMap } from '@admin/services/release/edit-release/footnotes/util';
+import {
+  Footnote,
+  FootnoteMeta,
+  FootnoteMetaGetters,
+} from '@admin/services/release/edit-release/footnotes/types';
+import { ErrorControlProps } from '@admin/validation/withErrorControl';
+import ReleaseFootnotesSection from '@admin/pages/release/edit-release/data/ReleaseFootnotesSection';
 import ReleaseDataUploadsSection from './ReleaseDataUploadsSection';
 import ReleaseFileUploadsSection from './ReleaseFileUploadsSection';
-import ReleaseFootnotesSection from './ReleaseFootnotesSection';
 
-const ReleaseDataPage = () => {
+export interface FootnotesData {
+  loading: boolean;
+  footnoteMeta: FootnoteMeta;
+  footnotes: Footnote[];
+  footnoteMetaGetters: FootnoteMetaGetters;
+  hasSufficientData: boolean;
+  canUpdateRelease: boolean;
+}
+
+const ReleaseDataPage = ({ handleApiErrors }: ErrorControlProps) => {
   const { publication, releaseId } = useContext(
     ManageReleaseContext,
   ) as ManageRelease;
 
+  const [footnotesData, setFootnotesData] = useState<FootnotesData>();
+  const [activeTab, setActiveTab] = useState<string>('');
+
+  const getFootnoteData = useCallback(() => {
+    Promise.all([
+      footnotesService.getReleaseFootnoteData(releaseId),
+      permissionService.canUpdateRelease(releaseId),
+    ])
+      .then(([{ meta, footnotes: footnotesList }, canUpdateReleaseResult]) => {
+        setFootnotesData({
+          loading: false,
+          footnoteMeta: meta,
+          footnotes: footnotesList,
+          footnoteMetaGetters: generateFootnoteMetaMap(meta),
+          hasSufficientData: !!Object.keys(meta).length,
```

Comment:
This can be calculated inside of `ReleaseFootnotesSection` itself, so seems redundant to have it here.